tests/test_makeps3iso.py

```
@@ -0,0 +1,281 @@
+    for _, value in pairs:
+        encoded = value.encode("utf-8") + b"\x00"
+        data_meta.append((len(data_blob), len(encoded)))
+        data_blob += encoded
+    num = len(pairs)
+    index_size = num * 16
+    key_table_off = 20 + index_size
+    data_table_off = key_table_off + len(keys_blob)
+    header = struct.pack("<4sHH", b"\x00PSF", 1, 1)
+    header += struct.pack("<III", key_table_off, data_table_off, num)
+    index = b""
+    for key_off, (data_off, data_len) in zip(key_offsets, data_meta):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)